packages/x-date-pickers/src/locales/beBY.ts

```diff
@@ -0,0 +1,77 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { DateView } from '../internals/models';
+
+const beBYPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Папярэдні месяц',
+  nextMonth: 'Наступны месяц',
+
+  // View navigation
+  openPreviousView: 'адкрыць папярэдні выгляд',
+  openNextView: 'адкрыць наступны выгляд',
+  calendarViewSwitchingButtonAriaLabel: (view: DateView) =>
+    view === 'year'
+      ? 'гадавы выгляд адкрыты, перайсці да каляндарнага выгляду'
+      : 'каляндарны выгляд адкрыты, перайсці да гадавога выгляду',
+  inputModeToggleButtonAriaLabel: (isKeyboardInputOpen, viewType) =>
+    isKeyboardInputOpen
+      ? `тэкставае поле адкрыта, перайсці да  ${viewType} выгляду`
+      : `${viewType} выгляд зараз адкрыты, перайсці да тэкставага поля`,
+
+  // DateRange placeholders
+  start: 'Пачатак',
+  end: 'Канец',
+
+  // Action bar
+  cancelButtonLabel: 'Адмена',
+  clearButtonLabel: 'Ачысціць',
+  okButtonLabel: 'OK',
+  todayButtonLabel: 'Сёння',
+
+  // Toolbar titles
+  datePickerToolbarTitle: 'Абраць дату',
+  dateTimePickerToolbarTitle: 'Абраць дату і час',
+  timePickerToolbarTitle: 'Абраць час',
+  dateRangePickerToolbarTitle: 'Абраць каляндарны перыяд',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Select ${view}. ${
```

Comment:
Same suggestion. Could you please refer to another locale (other than enUS) for guidance on how to translate `view` here.
As well as `Select` should probably be replaced by something else..? 🤔 

![Screenshot 2023-01-06 at 11 28 34](https://user-images.githubusercontent.com/4941090/210973527-e7713053-243c-433f-8250-539c62b8b31d.png)

Reply:
Thank you!